pages/admin/index.tsx

```diff
@@ -54,21 +54,24 @@ export default function Admin(props: AdminProps): JSX.Element {
             <AdminHeader>Dashboard</AdminHeader>
             <VStack spacing={5} mx={8} pb={4}>
                 <HStack spacing={4} w="100%">
-                    <AdminOptionButton
-                        icon={MdCreate}
-                        label="Create new Class"
-                        href="/admin/class"
-                    />
                     <AdminOptionButton
                         icon={MdClass}
                         label="Create new Program"
-                        href="/admin/program"
+                        href="/admin/program/edit/new"
+                    />
+                    <br></br>
+                    <AdminOptionButton
+                        icon={MdCreate}
+                        label="Create new Class"
+                        href="/admin/class/edit/new"
                     />
+                    <br></br>
                     <AdminOptionButton
-                        icon={MdPeople}
-                        label="Add Registrant"
-                        href="/admin/registrant"
+                        icon={MdPersonAdd}
+                        label="Add SDC Member"
+                        href="/admin/add"
                     />
+                    <br></br>
```

Comment:
A better fix for this weirdness is to fix the component itself. I fixed on my branch to refine the dashboard page but you can also do the same if you'd like:

In the `AdminOptionButton.tsx` component, the JSX element should be

```typescript
export const AdminOptionButton: React.FC<AdminOptionButtonProps> = ({
    icon,
    label,
    href,
    isExternal,
    ...props
}) => {
    return (
        <Link
            isExternal={isExternal}
            href={href}
            textDecoration="none"
            rounded={"md"}
            _active={{}}
            _hover={{}}
        >
            <Button
                border="2px"
                borderColor={colourTheme.colors.Sliver}
                px={9}
                py="24px"
                backgroundColor="white"
                _active={{}}
                _focus={{}}
                borderRadius={"6px"}
                fontWeight="normal"
                {...props}
            >
                <Icon mr={2} as={icon} />
                {label}
            </Button>
        </Link>
    );
};

rather than trying to wrap the link in another link.
```